### PR TITLE
Multi-worker ResNet Estimator benchmarks for PerfZero.

### PR DESCRIPTION
**Pull request by dubey**
This change refactors Resnet50EstimatorBenchmark into a base class and subclasses for single worker and multiple worker benchmarks.

### CODE REVIEW

File: official/resnet/estimator_benchmark.py [hunk 1]
```diff
@@ -342,6 +350,63 @@ def __init__(self, output_dir=None, root_data_dir=None, **kwargs):
...
 
+class Resnet50MultiWorkerEstimatorBenchmark(Resnet50EstimatorBenchmarkBase):
+  """Benchmarks for ResNet50 using Estimator with multiple workers."""
+
+  def __init__(self, output_dir=None, default_flags=None):
+    super(Resnet50MultiWorkerEstimatorBenchmark, self).__init__(
+        output_dir=output_dir,
+        default_flags=default_flags)
+
+  def benchmark_graph_fp16_8_gpu_ring_tweaked(self):
```
Comment (tfboyd) [inline official/resnet/estimator_benchmark.py]: How do we know how many workers and total GPUs used?

Comment (dubey) [reply official/resnet/estimator_benchmark.py]: The benchmark expects `TF_CONFIG` environment variable to be set with the list of workers and their IP addresses.

**Comment by tfboyd** [comment]
Cool let's keep moving forward and deal with reporting when we have the
first result.  I do not want to slow you down on something we can sort out
later and faster when it is running.  Merge when ready.

**Comment by guptapriya** [comment]
Thanks for the PR! Is there a plan to add real data and accuracy benchmark later on?

**Comment by dubey** [comment]
Yes, we'll expand the set of benchmarks and also include 2.0 tests.  I'd like to get the end to end setup working first.

**Pull request merged**